### ISSUE TITLE
How to change the batch size and the number of epochs when training?

### ISSUE THREAD

**Issue by HelloZjuLearner** [MrBlankness/LightM-UNet#18]
When use nnunetV2 before, the author told us to add a new class with the number of X epochs in nnunettrainer_Xepochs, and replace the nnunettrainer with nnunettrainer_Xepochs when training,does it still work in lightMunet? 
When changing batchsize , i went to the nnunetPlans.json, and it looks like going well , i would appreciate it if anybody could tell me how to change these two values.

**Comment by Hollycooool** [comment]
Have you solved this problem? I would like to know the reason why we "add a new class with the number of X epochs in nnunettrainer_Xepochs, and replace the nnunettrainer with nnunettrainer_Xepochs when training“

**Comment by HelloZjuLearner** [comment]
yes,you can straight go to the file "nnUnetTrain.py"(or similar name)，and change the "self.num_epochs" to your desired number.

"add a new class with the number of X epochs in nnunettrainer_Xepochs, and replace the nnunettrainer with nnunettrainer_Xepochs when training' is the method the author of nnunetV2 told us to do when changing epoch.








-----原始邮件-----
发件人:Hollycooool ***@***.***>
发送时间:2024-05-07 16:13:12 (星期二)
收件人: MrBlankness/LightM-UNet ***@***.***>
抄送: HelloZjuLearner ***@***.***>, Author ***@***.***>
主题: Re: [MrBlankness/LightM-UNet] How to change the batch size and the number of epochs when training? (Issue #18)






Have you solved this problem? I would like to know the reason why we "add a new class with the number of X epochs in nnunettrainer_Xepochs, and replace the nnunettrainer with nnunettrainer_Xepochs when training“
